src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -3532,7 +3552,7 @@ subroutine wrfv3_netcdf(fv3filenamegin)
 
     call gsi_copy_bundle(GSI_MetGuess_Bundle(it),gsibundle_fv3lam_dynvar_nouv) 
     call gsi_copy_bundle(GSI_MetGuess_Bundle(it),gsibundle_fv3lam_tracer_nouv) 
-    if( if_model_dbz )  call gsi_copy_bundle(GSI_MetGuess_Bundle(it),gsibundle_fv3lam_phyvar_nouv)
+    if( if_model_dbz .or. if_model_fed )  call gsi_copy_bundle(GSI_MetGuess_Bundle(it),gsibundle_fv3lam_phyvar_nouv)
```

Comment:
Will that be simpler and easier to undertand that only use test " if (allocated(gsibundle_fv3lam_phyvar_nouv)"?